demos/select.html

```diff
@@ -32,7 +32,7 @@
       }
 
       /* Hack to work around style-loader asynchronously loading styles. */
-      /* Equivalent to using mdc-typography's subheading2, which is used in the sass file. */
+      /* Equivalent to using mdc-typography's headline4, which is used in the sass file. */
```

Comment:
Have we tested whether this workaround is even still necessary? This might be from before we started referencing CSS directly in the demos...